mlir/test/mlir-pet/test_gemm.mlir

```diff
@@ -0,0 +1,55 @@
+ // RUN: ./../../../../bin/mlir-pet /llvm-project/mlir/tools/mlir-pet/inputs/gemm.c | FileCheck %s
```

Comment:
Can we avoid this?

Reply:
It is a relative path and it is awkward to fix as lit cannot cope with an absolute path. For now i dont know how to fix that.